blog/2015/06/training-lessons.html

```diff
@@ -0,0 +1,189 @@
+---
+layout: blog
+root: ../../..
+author: Greg Wilson
+title: "Training Lessons"
+date: 2015-06-22
+time: "13:00:00"
+category: ["Instructor Training"]
+---
+<!-- start excerpt -->
+<p>
+  I wrote about
+  <a href="{{page.root}}/blog/2015/05/online-instructor-training.html">our experiments with the format of instructor training</a>
+  back in May.
+  At that time,
+  we had run the class in three formats:
+</p>
+<ul>
+  <li>a multi-week online class,</li>
+  <li>an in-person two- or three-day class, and</li>
+  <li>the trainees physically together for two days with the trainer coming in via teleconference.</li>
+</ul>
+<p>
+  We have since tried it twice with the trainees at three different sites
+  (three universities in Arizona for one,
+  and universities in Cape Town, Sheffield, and Ann Arbor for the other).
+  We've also gathered
+  <a href="{{site.github_url}}/training-course/issues/514">a lot of feedback</a>
+  on what people want from instructor training
+  and <a href="{{site.github_url}}/training-course/issues/445">what its prerequisites should be</a>.
+  Here's what we've learned.
+</p>
+<!-- end excerpt -->
+<dl>
+  <dt><strong>The two-day format works as well as the online multi-week format, but in a different way.</strong></dt>
+  <dd>
+    <p>
+      The multi-week format gives people more time to read and reflect,
+      but squeezing everything into two days is more engaging.
+      I also think that people enjoy recording themselves and watching those videos more
+      (or dislike it less)
+      when they're together,
+      though that may actually reflect the fact that
+      they're doing training with people they know when they're co-located.
+    </p>
+  </dd>
+  <dt><strong>The online two-day format is as good as the in-person two-day format.</strong></dt>
```

Comment:
"seems to be" rather than "is"?  (All I've got is end-of-class feedback, 
and I haven't noticed a difference between the two formats...)